rero_ils/modules/acq_order_lines/jsonschemas/acq_order_lines/acq_order_line-v0.0.1.json

```diff
@@ -336,6 +295,11 @@
           "pattern": "^https://bib.rero.ch/api/libraries/.*?$"
         }
       }
+    },
+    "cancelled": {
+      "title": "Cancelled",
```

Comment:
add a description to explain the field .. "If checked this order line should be considerate as cancelled" or something like this